bc_obps/Makefile

```diff
@@ -3,7 +3,7 @@ PSQL=psql -h localhost
 DB_NAME=registration
 MANAGE_PY = manage.py
 SCHEMA_NAME=erc
-PYTEST=pytest --rootdir=bc_obps
+PYTEST=pytest --rootdir=bc_obps --import-mode=importlib
```

Comment:
n00b question: what does `--import-mode=importlib` do?

Reply:
This is actually an advanced question! 😃 Here’s the relevant documentation:  
[Pytest PythonPath Explanation](https://docs.pytest.org/en/7.1.x/explanation/pythonpath.html#:~:text=modules%20after%20importing.-,importlib,-%3A%20new%20in%20pytest)  

This introduces a new approach that helps us avoid adding an `__init__.py` file in every test directory just to recognize test files.